python/py-pyside2/Portfile

```diff
@@ -16,19 +16,25 @@ homepage                https://wiki.qt.io/Qt_for_Python
 license                 {GPL-2 GPL-3+ LGPL-3}
 master_sites            https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${version}-src
 distname                pyside-setup-opensource-src-${version}
-use_xz                  yes
```

Comment:
this should stay since that latest version again provides a `.tar.xz` file and these archives are smaller

Reply:
👍🏻